modules/statics/src/networks.ts

```diff
@@ -712,15 +712,13 @@ class XrpTestnet extends Testnet implements AccountNetwork {
 class Xtz extends Mainnet implements AccountNetwork {
   name = 'Xtz';
   family = CoinFamily.XTZ;
-  explorerUrl = 'https://tezblock.io/transaction/';
```

Comment:
I'm not sure what to use for the account explorer url, based on what I see I think its just the base url with an account id appended